CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,70 @@
+# Tinted Theming Code of Conduct
+
+## Our Commitment
+
+Tinted Theming is committed to providing a welcoming and inclusive community where everyone feels respected and valued. We expect all community members, including contributors, maintainers, and users, to adhere to this Code of Conduct.
+
+## Inclusivity and Respect
+
+We value diversity and believe that a wide range of perspectives and experiences enriches our community. We are committed to creating an environment free from discrimination, harassment, and disrespectful behavior.
+
+## Language and Communication
+
+We strive for a community where everyone communicates with respect and empathy. Language has power, and we must use it responsibly.
+
+### Understanding Slurs and Offensive Language
+
+It's important to be aware that certain words, even those that might seem commonplace, can be harmful. This includes slurs or derogatory language directed at any disadvanged and underrepresented group, whether based on race, ethnicity, gender, disability, sexual orientation, or other characteristics. Such language:
+
+- Contributes to a culture of exclusion.
+- Creates an unwelcoming environment for affected individuals and groups.
+- Contradicts our values of respect and inclusivity.
+
+## Expected Behavior
+
+We expect all members of the Tinted Theming community to:
```

Comment:
One additional one I'd like to see is something about trying to assume good/positive intentions. Believe the best of people and if something seems off, try to clarify before jumping to conclusions about what someone else is thinking.

Reply:
Yeah that's important, I've added it as the first point.